subsys/spm/Kconfig

```diff
@@ -153,6 +153,10 @@ config SPM_NRF_PWM3_NS
 	bool "PWM3 is Non-Secure"
 	default y
 
+config SPM_NRF_REGULATORS_NS
+	bool "Regulators is Non-Secure"
+	default y
```

Comment:
looks good, I believe the default should be 'n' (so just remove default configuration assessment).

Reply:
Thank you for the comment. I'll update in a new commit.

Reply:
I committed the update then found that not able to set this flag from application project file. Is there any way to do so?

Reply:
To set this flag you need to modify the `nrf/samples/nrf9160/spmprj.conf` since this config is not for the application image, only for the spm image.

Reply:
Confirmed yes in the .config of spm.
This would have global affect to all samples, just like default 'y' in spm Kconfig, though.

Reply:
@junqingzou It's possible to set it on a project-basis with a few lines in `CMakeLists.txt`, see [here]( https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/samples/nrf9160/lte_ble_gateway/CMakeLists.txt).

Reply:
@lemrey great tip, thanks! I copied the solution, and just by coincidence, my project needs to use UARTE2, too. So put the two in Non-secure in the child spm file.